lib/integration/scenarios/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -1759,10 +1745,9 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             fromIntegral (unCoin c)
 
     costOfJoining :: Context -> Natural
-    costOfJoining ctx =
-        if _mainEra ctx >= ApiBabbage
-            then costOf (TxSize 485) ctx
-            else costOf (TxSize 479) ctx
+    costOfJoining ctx = if _mainEra ctx >= ApiConway
+            then costOf (TxSize 519) ctx -- guess: 32+2 bytes more for drep
```

Comment:
Hm  - will need to see if this works

Reply:
> 491 appears to be the right value

Locally, but not in CI — I think this is the expiration slot dependent flakiness 😢 . Trying `491 + 4` instead (5 vs 9 byte cbor integer)